cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/MultiLiveStreamFileReader.java

```diff
@@ -85,7 +85,7 @@ public int read(Collection<? super StreamEventOffset> events, int maxEvents,
                   long timeout, TimeUnit unit, ReadFilter readFilter) throws IOException, InterruptedException {
     int eventsRead = 0;
 
-    Stopwatch stopwatch = new Stopwatch();
+    Stopwatch stopwatch = Stopwatch.createStarted();
     stopwatch.start();
```

Comment:
@bipinsoniguavus The `stopwatch` you created, has stared by default(since you used `createStarted` function), so we can remove this line.

Reply:
Fix at other places also if applicable.

Reply:
Done